userguide/dailyuse/anbox.rst

```diff
@@ -26,7 +26,7 @@ How to install
 .. warning::
     Installing Anbox is only recommended for experienced users.
 
-- Make sure your supported device runs on 16.04 (Anbox doesn't work on 15.04)
+- Make sure your supported device runs on 16.04 (Anbox doesn't work on 15.04); be sure to have a `_backup <../backup.rst>`_ of the device.
```

Comment:
The link doesn't seem to go anywhere. Do we have a backup doc? 

https://ci.ubports.com/job/docs.ubports.com/job/PR-258/3/artifact/_build/html/userguide/dailyuse/anbox.html

Also, there is some extra underscore.

Lastly. maybe it could be it's own bullet point?